content/en/docs/concepts/containers/images.md

```diff
@@ -47,6 +47,20 @@ to roll back to a working version.
 Instead, specify a meaningful tag such as `v1.42.0`.
 {{< /caution >}}
 
+### ImagePullBackOff
+When a kubelet starts creating containers for a Pod using a container runtime, 
+it might be possible the container is in [Waiting](/docs/concepts/workloads/pods/pod-lifecycle/#container-state-waiting) 
+state because of `ImagePullBackOff`.
+
+The status `ImagePullBackOff` means that a container could not start because Kubernetes 
+could not pull a container image (for reasons such as invalid image name, or pulling 
+from a private registry without `imagePullSecret`). The `BackOff` part indicates 
+that Kubernetes will keep trying to pull the image, with an increasing back-off delay.
+
+{{< note >}}
+The maximum image pull back off is `300 seconds`.
+{{< /note >}}
```

Comment:
I'd write
```suggestion
Kubernetes raises the delay between each attempt until it reaches a compiled-in limit,
which is 300 seconds (5 minutes).
```

Reply:
Thanks for the review @sftim , I have made the changes.